common/src/main/cpp/extensions/openxr_fb_face_tracking_extension_wrapper.cpp

```diff
@@ -113,14 +112,14 @@ void OpenXRFbFaceTrackingExtensionWrapper::_on_session_created(uint64_t instance
 	}
 
 	// Create the face-tracker handle
-    XrFaceTrackingDataSource2FB dataSources[2] = {
-        XR_FACE_TRACKING_DATA_SOURCE2_VISUAL_FB,
-        XR_FACE_TRACKING_DATA_SOURCE2_AUDIO_FB
-    };
+	XrFaceTrackingDataSource2FB dataSources[2] = {
+		XR_FACE_TRACKING_DATA_SOURCE2_VISUAL_FB,
+		XR_FACE_TRACKING_DATA_SOURCE2_AUDIO_FB
+	};
 	XrFaceTrackerCreateInfo2FB createInfo2 = { XR_TYPE_FACE_TRACKER_CREATE_INFO2_FB };
 	createInfo2.faceExpressionSet = XR_FACE_EXPRESSION_SET2_DEFAULT_FB;
-    createInfo2.requestedDataSourceCount = 2;
-    createInfo2.requestedDataSources = dataSources;
+	createInfo2.requestedDataSourceCount = 2;
+	createInfo2.requestedDataSources = dataSources;
```

Comment:
We're standardizing on using braced-initialization for enums initialization. See https://github.com/GodotVR/godot_openxr_vendors/pull/100#issuecomment-1981726339 for reference.

Reply:
Switched to brace initialization.